src/components/general/navbar/nav-lists/NavApplications.tsx

```diff
@@ -0,0 +1,123 @@
+                            ? "text-xs duration-300"
+                            : "text-xs opacity-50 duration-300"
+                        }
+                      >
+                        Check OD after mod change
+                      </p>
+                    </div>
+                  </div>
+                </Link>
+              </NavigationMenuLink>
+            </li>
+            <li>
+              <NavigationMenuLink asChild>
+                <Link href="/profile-adjustment" className="hover:no-underline">
+                  <div
+                    className="w-64 p-3 duration-200 rounded-sm flex gap-3 place-items-center"
+                    onMouseEnter={() => setApp(2)}
+                    onMouseLeave={() => setApp(0)}
+                  >
+                    <MagnifyingGlassIcon
+                      className={
+                        app === 2
+                          ? "h-6 rounded-sm text-black bg-white duration-300"
+                          : "h-6 rounded-sm duration-300"
+                      }
+                    />
+                    <div className="flex flex-col">
+                      <p className="font-bold text-sm">Profile Adjustment</p>
+                      <p
+                        className={
+                          app === 2
+                            ? "text-xs duration-300"
+                            : "text-xs opacity-50 duration-300"
+                        }
+                      >
+                        Explore players&apos; profile
+                      </p>
+                    </div>
+                  </div>
+                </Link>
+              </NavigationMenuLink>
+            </li>
+            <li>
+              <NavigationMenuLink asChild>
+                <Link href="/pp" className="hover:no-underline">
+                  <div
+                    className="w-64 p-3 duration-200 rounded-sm flex gap-3 place-items-center"
+                    onMouseEnter={() => setApp(3)}
+                    onMouseLeave={() => setApp(0)}
+                  >
+                    <CalculatorIcon
+                      className={
+                        app === 3
+                          ? "h-6 rounded-sm text-black bg-white duration-300"
+                          : "h-6 rounded-sm duration-300"
+                      }
+                    />
+                    <div className="flex flex-col">
+                      <p className="font-bold text-sm">PP App</p>
+                      <p
+                        className={
+                          app === 3
+                            ? "text-xs duration-300"
+                            : "text-xs opacity-50 duration-300"
+                        }
+                      >
+                        Calculate the PP on a map
+                      </p>
+                    </div>
+                  </div>
+                </Link>
+              </NavigationMenuLink>
+            </li>
```

Comment:
her ser vi gjentakelse av mye av det samme inni <li>...</li>. Det kan være verdt å:

Lage en NavApplicationsItem component som tar props: icon, description, title, (app og index også)

Lage en variabel `apps = [{icon: CalculatorIcon, description: "Calculate the PP on a map", title: "PP app"}, ...]` også loope over den:

`apps.map((currentApp, index) => {return <NavApplicationsItem {...currentApp} app={app} index={index} />})`

Reply:
boom